test/lcevc/lcevc_integration.js

```diff
@@ -0,0 +1,101 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+describe('LCEVC Integration', () => {
+  const Util = shaka.test.Util;
+
+  /** @type {!jasmine.Spy} */
+  let onErrorSpy;
+
+  /** @type {!HTMLCanvasElement} */
+  let canvas;
+
+  /** @type {!HTMLVideoElement} */
+  let video;
+  /** @type {shaka.Player} */
+  let player;
+  /** @type {!shaka.util.EventManager} */
+  let eventManager;
+
+  let compiledShaka;
+
+  /** @type {!shaka.test.Waiter} */
+  let waiter;
+
+  const seiManifests = {
+    FMP4_DASH: '/base/test/test/assets/lcevc-sei/lcevc-sei.mpd',
+  };
+
+  beforeAll(async () => {
+    video = shaka.test.UiUtils.createVideoElement();
+    canvas = shaka.test.UiUtils.createCanvasElement();
+
+    document.body.appendChild(video);
+    document.body.appendChild(canvas);
+
+    compiledShaka =
+        await shaka.test.Loader.loadShaka(getClientArg('uncompiled'));
+  });
+
+  beforeEach(async () => {
+    await shaka.test.TestScheme.createManifests(compiledShaka, '_compiled');
+    player = new compiledShaka.Player();
+
+    player.attachCanvas(canvas);
+    await player.attach(video);
+
+    // Enable the LCEVC enhancement.
+    player.configure('lcevc.enabled', true);
+    player.configure('lcevc.drawLogo', true);
+    player.configure('lcevc.dynamicPerformanceScaling', false);
+
+    // Enable transmuxing
+    player.configure('streaming.preferNativeHls', false);
+    player.configure('mediaSource.forceTransmux', true);
```

Comment:
It is not necessary, so it should be removed. TS already does transmux by default, and we also prefer MSE over HLS by default.